Jasmine/Common/Extensions/CGPoint+Arithmetic.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+import UIKit
+
+extension CGPoint {
+
+    /// Subtracts another point to get a displacement vector.
+    ///
+    /// - Parameter other: the other point to get displacement.
+    /// - Returns: the resultant vector
+    func sub(_ other: CGPoint) -> CGVector {
```

Comment:
Subtract two point gives you a vector mah. Don't make sense fort it to be a point. But let me check if this code is needed anymore